samples/bluetooth/peripheral_lbs/child_image/mcuboot/boards/thingy53_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
Does it need to be copied to other Matter samples as well? What has changed that it used to not be necessary?